packages/client/test/rpc/engine/newPayloadV4.spec.ts

```diff
@@ -138,6 +143,11 @@ const electraGenesisContracts = {
     nonce: '1',
     code: '0x3373fffffffffffffffffffffffffffffffffffffffe1460575767ffffffffffffffff5f3511605357600143035f3511604b575f35612000014311604b57611fff5f3516545f5260205ff35b5f5f5260205ff35b5f5ffd5b5f35611fff60014303165500',
   },
+  '0x00b42dbF2194e931E80326D950320f7d9Dbeac02': {
+    balance: '0',
+    nonce: '1',
+    code: '0x3373fffffffffffffffffffffffffffffffffffffffe146098573615156028575f545f5260205ff35b36606014156101445760115f54600182026001905f5b5f82111560595781019083028483029004916001019190603e565b90939004341061014457600154600101600155600354806004026004013381556001015f35815560010160203581556001016040359055600101600355005b6003546002548082038060011160ac575060015b5f5b81811460f15780607402838201600402600401805490600101805490600101805490600101549260601b84529083601401528260340152906054015260010160ae565b9101809214610103579060025561010e565b90505f6002555f6003555b5f548061049d141561011d57505f5b6001546001828201116101325750505f610138565b01600190035b5f555f6001556074025ff35b5f5ffd',
```

Comment:
Is this the code for this new system contract to perform consolidations? Can we provide some minimal commentary on how it works?

Reply:
yes, its fully explained in EIP in detail